src/sign-in-webapp/README.md

```diff
@@ -1,52 +1,55 @@
 
-Later the web app can make calls to the Microsoft 365 Graph API on behalf of signed-in user.
-```
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
 
-:link: For more information about how to proctect your projects, please let's take a look at https://docs.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code. To know more about how this sample has been generated, please visit https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/?view=aspnetcore-6.0
+Use the following settings for your app registration:
 
-## Prerequisites
+| App registration <br/> setting | Value for this sample app                          | Notes                                                                                                       |
+|:------------------------------:|:---------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+| **Name**                       | `active-directory-dotnet-webapp-aspnetcore`        | Suggested value for this sample. <br/> You can change the app name at any time.                             |
+| **Supported account types**    | **My organization only**                           | Required for this sample. <br/> Support for the Single tenant.                                              |
+| **Platform type**              | `Web`                                              | Required value for this sample. <br/> Enables the required and optional settings for the app type.          |
+| **Redirect URIs**              | `https://localhost:5001/signin-oidc`               | Required value for this sample. <br/> You can change that later in your own implementation.                 |
+| **Front-channel logout URL**   | `https://localhost:5001/signout-oidc`              | Required value for this sample. <br/> You can change that later in your own implementation.                 |
+| **Client secret**              | _Value shown in Azure portal_                      | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).                 |
 
-1. An Azure Active Directory (Azure AD) tenant. You can [open an Azure account for free](https://azure.microsoft.com/free) to get an Azure AD instance.
-- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box or select in the Azure portal.
 
-## Register the web API application in your Azure Active Directory
+<details>
+   <summary>:computer: Alternative: Register the application using az-cli</summary>
 
 1. Choose a password for the Azure AD App
```

Comment:
Not required (we can do this sort of thing in a content pass) but be super explicit in procedural steps.

~~~suggestion
1. Set a shell environment variable containing a client secret for the app (for the `--password` argument in the next step):

    ```bash
   export AZURE_AD_APP_SECRET=<at-least-sixteen-characters-here>
    ```
~~~

> NOTE: Not able to suggest change to the unchanged code fence lines. :(

Reply:
thanks for the feedback @mmacy 

done | addressed from [5b0f6](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/9/commits/5b0f6b46805ece260994496c08d2d709440cd508) and [22736](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/9/commits/227368d8a3cdc26a46527c46da65e7724565f29a)